defs/f256vtio.d

```diff
@@ -30,7 +30,13 @@ V.FBCol             RMB       1                   currently selected foreground
 V.BordCol           RMB       1                   currently selected border color
 V.KeyDrvMPtr        RMB       2                   keydrv module address
 V.KeyDrvEPtr        RMB       2                   keydrv entry point address
-
+		    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU.  Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.